arctic/chunkstore/date_chunker.py

```diff
@@ -74,7 +74,17 @@ def to_start_end(self, data):
         return self._get_date_range(data)
 
     def to_mongo(self, range_obj):
-        return range_obj.mongo_query()
+        if range_obj[0] and range_obj[1]:
+            return {'$and': [{'start': {'$lte': range_obj[1]}}, {'end': {'$gte': range_obj[0]}}]}
```

Comment:
Would be good to do an `.explain()` on these queries... I suspect Mongo won't do anything efficient with a query that contains a `start` and `end` like this